docker/jaas/webclient/start.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# Docker variables
+IMAGE_NAME="jaas-web-client"
+IMAGE_TAG="1.2.6"
+CONTAINER_NAME="jwc"
+HTTP_HOST_PORT=8080
+
+# Container env variables
+JWC_HTTP_PORT=80
+JWC_JDBC_DRIVER=com.mysql.jdbc.Driver
+JWC_JDBC_URL=jdbc:mysql://172.18.128.228:3306/jaggerdb
+JWC_JDBC_USER=root
+JWC_JDBC_PASS=root
+
+# Step 1. Check and build the image if needed.
+echo "'Check and build the image if needed' phase..."
+is_image_created=$(docker images | grep ${IMAGE_NAME})
+if [ -z "${is_image_created}" ]; then
+    echo "Image ${IMAGE_NAME} is not built yet. Building..."
+    cp -v ${HOME}/.m2/repository/com/griddynamics/jagger/webclient/1.2.6-SNAPSHOT/webclient-1.2.6-SNAPSHOT-exec-war.jar \
```

Comment:
It's better to extract version to var, since it's gonna be changing all the time.